client/app/components/ParameterValueInput.less

```diff
@@ -0,0 +1,19 @@
+@import '~antd/lib/input-number/style/index'; // for ant @vars
+
+.parameter-apply-button {
+  margin-left: -55px;
```

Comment:
This is the cause of the FF display issue.
It's better to take the button "out of the flow" so it doesn't affect neighboring elements.

```css
parameter-value-input {
  position: relative;
  display: inline-block;
}

.parameter-apply-button {
  width: 50px;
  position: absolute !important;
  right: 4px;
  top: 6px; /* 5px after button height set to 25px */
}
```